packages/x-date-pickers/src/DesktopDateTimePicker/tests/describes.DesktopDateTimePicker.test.tsx

```diff
@@ -35,6 +35,7 @@ describe('<DesktopDateTimePicker /> - Describes', () => {
     defaultProps: {
       views: ['day'],
       openTo: 'day',
+      closeOnSelect: true,
```

Comment:
This becomes a necessity if we go with the logic of `closeOnSelect: false` (if we are showing multi-section digital clock.)
If we do go this route: Should we maybe also add logic for this default to be active if `views` contains any `isTimeView`? 🤔

Reply:
After thinking about it more, I've defaulted to adding `accept` action given that a multi-section digital clock would be shown.